backend/src/LeanCode.AppRating/Processes/SendEmailOnLowRateSubmitted.cs

```diff
@@ -0,0 +1,46 @@
+using LeanCode.AppRating.Configuration;
+using LeanCode.AppRating.EmailViewModels;
+using LeanCode.SendGrid;
+using MassTransit;
+using SendGrid.Helpers.Mail;
+using Serilog;
+
+namespace LeanCode.AppRating.Processes;
+
+public class SendEmailOnLowRateSubmitted<TUserId> : IConsumer<RateSubmitted<TUserId>>
+    where TUserId : notnull, IEquatable<TUserId>
+{
+    private readonly ILogger logger = Log.ForContext<SendEmailOnLowRateSubmitted<TUserId>>();
+    private readonly SendGridRazorClient sendGridRazorClient;
+    private readonly AdminConfiguration adminConfiguration;
+
+    public SendEmailOnLowRateSubmitted(SendGridRazorClient sendGridRazorClient, AdminConfiguration adminConfiguration)
+    {
+        this.sendGridRazorClient = sendGridRazorClient;
+        this.adminConfiguration = adminConfiguration;
+    }
+
+    public async Task Consume(ConsumeContext<RateSubmitted<TUserId>> context)
+    {
+        if (context.Message.Rating > adminConfiguration.MaximalLowRatingInclusive)
+        {
+            logger.Debug("Rating is not considered low, skipping");
+            return;
+        }
+
+        var vm = new LowRateSubmittedEmail { UserId = context.Message.UserId.ToString() };
+
+        var message = new SendGridLocalizedRazorMessage("en")
+            .WithSubject("emails.low-rate-submitted.subject")
```

Comment:
I'm afraid this won't be easily discoverable by the consumers of this package. And because making this externally visible in code isn't too useful because it needs to be used in RESX, maybe make this (and culture passed just above while you're at it) configurable as well?